src/modV.js

```diff
@@ -258,6 +259,49 @@ class ModV extends EventEmitter2 {
 
 		return layerIndex;
 	}
+
+	/** @return {ModV.WorkersDataType} */
+	createWorkers() {
+		const palette = new PaletteWorker();
+		palette.on(PaletteWorker.EventType.PALETTE_ADDED, this.paletteAddHandler.bind(this));
+		palette.on(PaletteWorker.EventType.PALETTE_UPDATED, this.paletteUpdateHandler.bind(this));
+
+		return {
+			palette,
+		};
+	}
+
+	/**
+	 * @protected
+	 * @param {string} data
+	 */
+	paletteAddHandler(id) {
+		if(!this.palettes.has(id)) {
+			this.palettes.set(id, {});
+		} else {
+			console.error('Palette with ID', id, 'already exists');
+		}
+	}
+
+	/**
+	 * @protected
+	 * @param {string} id
+	 * @param {*} currentColor
+	 * @param {*} currentStep
+	 * @todo Types
+	 */
+	paletteUpdateHandler(id, currentColor, currentStep) {
+		this.palettes.set(id, {
+			currentColor: currentColor,
+			currentStep: currentStep
+		});
+	}
+
+	/** @param {string} id */
+	removePalette(id) {
```

Comment:
I think that splitting class definition (by keep some prototype methods in different files) is a bad idea, because preferring modular architecture – one thing (one functionality) = one module. If we want to split main class (modV) into modules – we should use encapsulation, not just splitting into files. In further we definitely should do more work to abstract this functionality (for example, we can later create some method in modV constructor like `registerWorkerModule` for abstract module registration, move this handlers to this module, and replace this handlers with more abstract handlers, no palette-specific). But at this moment its implicit, IMHO.
How do you like that?